src/current/v23.1/architecture/glossary.md

```diff
@@ -41,3 +41,220 @@ For more information on deployment options and guidelines on how to choose a dep
+
+<a name="cartographic-projection"></a>
+
+- _Cartographic projection_: A cartographic projection, or map projection, is the process used to represent 3-dimensional (or higher) data on a 2-dimensional surface. This is usually related to how we might display 3-dimensional shapes represented in a database by the [`GEOGRAPHY` data type](#geography) on the surface of a map, which is a flat plane. For more information, see the GIS Lounge article [What is a Map Projection?](https://www.gislounge.com/map-projection/) by Caitlin Dempsey.
+
+<a name="covering"></a>
+
+- _Covering_: The covering of a shape _A_ is a set of locations (in CockroachDB, [S2 cell IDs](#s2)) that comprise another shape _B_ such that no points of _A_ lie outside of _B_.
+
+<a name="geocoder"></a>
+
+- _Geocoder_: Takes an address or the name of a place, and returns latitude and longitude coordinates. For more information, see [the wikipedia article on Geocoding](https://en.wikipedia.org/wiki/Geocoding).
+
+<a name="nearest-neighbor-search"></a>
+
+- _Nearest-neighbor search_: Given a starting point on a map and a set of search criteria, find the specified number of points nearest the starting point that meet the criteria. For example, a nearest-neighbor search can be used to answer the question, "What are the 10 closest [Waffle House restaurants](https://www.wafflehouse.com) to my current location?"  This is also sometimes referred to as "_k_ nearest-neighbor" search.
+
+<a name="srid"></a>
+
+- _SRID_: The Spatial Referencing System Identifier (a.k.a. SRID) is used to tell which spatial reference system will be used to interpret each spatial object. A [commonly used SRID is 4326](../srid-4326.html), which represents spatial data using longitude and latitude coordinates on the Earth's surface as defined in the [WGS84](#wgs84) standard.
+
+<a name="spatial-reference-system"></a>
+
+- _Spatial reference system_: Used to define what a spatial object "means". For example, a spatial object could use [geographic](#geography) coordinates using latitude and longitude, or a [geometry](#geometry) projection using points with X,Y coordinates in a 2-dimensional plane.
+
+### Data types
+
+<a name="geometry"></a>
+
+- `GEOMETRY`: Used to represent shapes relative to 2-, 3-, or higher-dimensional plane geometry.  For more information about the spatial objects used to represent geometries, see:
+  - [`POINT`](../point.html)
+  - [`LINESTRING`](../linestring.html)
+  - [`POLYGON`](../polygon.html)
+  - [`MULTIPOINT`](../multipoint.html)
+  - [`MULTILINESTRING`](../multilinestring.html)
+  - [`MULTIPOLYGON`](../multipolygon.html)
+  - [`GEOMETRYCOLLECTION`](../geometrycollection.html)
+
+<a name="geography"></a>
+
+- `GEOGRAPHY`: Used to represent shapes relative to locations on the Earth's [spheroidal](#spheroid) surface.
+
+### Data formats
+
+<a name="ewkt"></a>
```

Comment:
This looks like it's in the wrong place (should be above EWKT).

Reply:
Fixed this by moving it to a H4